nodes/text/change_text_case.py

```diff
@@ -28,4 +29,6 @@ def draw(self, layout):
         layout.prop(self, "caseType", text = "")
 
     def getExecutionCode(self):
+        if self.caseType == "CAPITALIZE_WORDS":
+            return "outText = ' '.join( [t.capitalize() for t in inText.split()])"
```

Comment:
Maybe we can have two options here... check this: http://stackoverflow.com/a/1549644/4755171
(also the comments below this post)